free-programming-books-ar.md

```diff
@@ -32,39 +39,55 @@
 * [الإدارة المتقدمة لجنو/لينكس ](http://librebooks.org/gnu-linux-advanced-administration/)
 * [دفتر مدير دبيان](http://librebooks.org/debian-handbook-arabic/)
 * [دليل إدارة خواديم أوبنتو ](http://librebooks.org/ubuntu-server-guide-arabic/)
+* [كتاب لينكس الشامل](https://www.kutub.info/library/book/596)
+* [سطر اوامر لينكس](111000.net/systems/files/The_Linux_Command_Line-arabic-14.07.pdf) (PDF)
+* [يوميات لينكساوي - الجزء الأول](http://librebooks.org/linuxawy-diaries-1/)
+* [يوميات لينكساوي - الجزء الثاني](http://librebooks.org/linuxawy-diaries-2/)
 
 
-### open source software
+### Open Source Software
 
 * [الأبعاد الاستراتيجية للبرمجيات الحرة مفتوحة المصدر ](http://librebooks.org/strategic-dimensions-of-free-and-open-source-software/)
 * [الحريات الرقمية - المفاهيم الرئيسية ](http://librebooks.org/digital-freedoms-main-concepts/)
 * [المصادر المفتوحة خيارات بلا حدود ](http://librebooks.org/opensource-ultimate-options/)
 * [تعرف على البرمجيات الحرة ](http://librebooks.org/know-free-software/)
-* [دليل البرمجيات الحرة مفتوحة ](http://librebooks.org/free-opensource-guide/)
+* [دليل البرمجيات الحرة مفتوحة المصدر](http://librebooks.org/free-opensource-guide/)
 * [نبذة عن رخص البرمجيات الحرة ](http://librebooks.org/bref-about-foss-licenses/)
+* [دليل المستخدم في ليبر أوفيس](http://librebooks.org/libreoffice-arabic-manual/)
+* [الرسم في جيمب](http://librebooks.org/drawing-in-gimp/)
 
 
 ### Operating Systems
 
 * [المقدمة في تحليل وتصميم أنظمة ](http://librebooks.org/intro-to-os-analysis-and-design/)
+* [مفاهيم نظم التشغيل](https://www.kutub.info/library/book/12880)
+* [برمجة أنظمة التشغيل](https://www.kutub.info/library/book/10357)
 
 
-### php
+### PHP
 
 * [تعلم البرمجة بلغة PHP](http://librebooks.org/learn-programming-with-php/)
 
 
+### Python
+
+* [خطوة على طريق بايثون](http://www.linuxac.org/forum/memfiles/byteofpython_arabic.pdf) (PDF)
+* [تعلم البرمجة مع باثون 3](https://www.kutub.info/library/book/18842)
```

Comment:
kutub.info looks like a file-hosting site of the sort we don't permit. For example, the proper source for this book appears to be http://inforef.be/swi/download/apprendre_python3_arab.pdf  It's a good practice to add the original author to help distinguish books with similar titles.